src/engine/N3Base/N3FXBundle.h

```diff
@@ -37,6 +37,8 @@ class CN3FXBundle : public CN3BaseFileAccess {
     int m_iSourceJoint;
 
     bool m_bDependScale;
+    bool m_bEarthQuake;             // Needs Implementation
```

Comment:
Based on the leaked project files, which these names are apparently based off, this is in fact definitely a bool back then, but later versions do still show the same property reporting integral values instead of true/false.

So it *was* definitely originally a bool.

Reply:
I now understand after reading all your replies repeatedly and why you are saying it. Like I said back then IDK and you're probably right. You are right since I've never touched new files and going after everything again I understand what you were saying, even though I don't support those newer FX you are talking about nor have parsed them successfully. I only had access to the newer earthquake FX mostly mage skills. Thanks for your input and excuse me for my stubbornness. You are right